### ISSUE TITLE
Allow graph view to be embedded into html

### ISSUE THREAD

**Issue by wzsRQQjaYN** [KosmosisDire/obsidian-webpage-export#131]
Is there a way to export a standalone HTML page with the global graph in it?

**Comment by KosmosisDire** [comment]
No, as it says in the description this is not supported. This is because it uses wasm which is complicated to embed into html. I plan on eventually supporting this, but for now you can't. Thanks!

**Comment by candideu** [comment]
> No, as it says in the description this is not supported. 

Is this still the case @KosmosisDire? I tried finding notes on this in the README and docs, but didn't see anything.
